addons/medical/CfgEden.hpp

```diff
@@ -9,8 +9,8 @@ class Cfg3DEN {
             };
         };
         class GVAR(isMedicControl): Title {
-            attributeLoad = "(_this controlsGroupCtrl 100) lbsetcursel (((_value + 1) min 3) max 0);";
-            attributeSave = "(missionnamespace getvariable ['ace_isMedic_temp',0]) - 1;";
+            attributeLoad = "(_this controlsGroupCtrl 100) lbSetCurSel (((_value + 1) min 3) max 0);";
+            attributeSave = "(_this getVariable ['ace_isMedic_temp', 0]) - 1;";
```

Comment:
That's how BIS does attributes:
```
class Rank: Toolbox {
    attributeLoad = "(_this controlsGroupCtrl 100) lbsetcursel (['private','corporal','sergeant','lieutenant','captain','major','colonel'] find _value);";
    attributeSave = "['private','corporal','sergeant','lieutenant','captain','major','colonel'] select (missionnamespace getvariable ['Rank_value',0]);";
    class Controls: Controls {
        class Value: Value {
            tooltips[] = {"Private", "Corporal", "Sergeant", "Lieutenant", "Captain", "Major", "Colonel"};
            onToolboxSelChanged = "missionnamespace setvariable ['Rank_value',_this select 1];";
        };
    };
};
```
Does rank have the same issue as ace's medic setting?